packages/renderer/src/lib/images/WindowsMinIcon.svelte

```diff
@@ -1,5 +1,6 @@
 <script lang="ts">
 export let size = 16;
+export let color = '#ffffff';
```

Comment:
I'm wondering why this is not from color-registry and hard coded ?

Reply:
This is currently hard-coded in the icon, this allows to change the text color. But I can try to change it to use the color-registry ones